src/components/NavBar.tsx

```diff
@@ -0,0 +1,56 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Container, Nav, Navbar, Dropdown } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars } from "@fortawesome/free-solid-svg-icons";
+import { RoutePaths } from "../common/Routes";
+
+interface NavBarProps {
+  links: { text: string; href: string }[];
+}
+
+export function NavBar({ links }: NavBarProps) {
+  return (
+    <Navbar expand="lg" fixed="top" className="custom-navbar">
+      <Container fluid>
+        <Navbar.Brand href={RoutePaths.DASHBOARD} className="brand-text">
+          <img
+            src="public/images/logo2.png"
+            alt="Logo"
+            style={{ height: "40px", marginBottom: "10px" }}
+          />
+          OralCheckr
+        </Navbar.Brand>
+        <Dropdown className="ms-auto d-lg-none">
+          <Dropdown.Toggle
+            as="span"
+            id="dropdown-basic"
+            className="dropdown-icon"
+          >
+            <FontAwesomeIcon icon={faBars} />
+          </Dropdown.Toggle>
+
+          <Dropdown.Menu align="end">
```

Comment:
I am confused where is the dropdown being used in the NavBar?

Reply:
When the screen is smaller, the links in the navbar change to a dropdown style to fit better. If you think they should stay in the navbar as is for smaller screens, i can get rid of the dropdown.

Reply:
I see. That makes sense! 
To keep things simple, I'd suggest let's keep only one navbar, either on the left side or on the top.
If in the future, we think we need two navbars, we can always add that.

What I think we can do is have the following options:
- Questionnaire
- Recommendations (this page can show the dashboard and the oral health status)
- Habit Tracker

What do you think?

Reply:
Ok let's do one navbar on the top. Those suggestions for the options sound good. I'll make those changes.